ItemLevelDownloader/Providers/teamcity.handlebars

```diff
@@ -0,0 +1,25 @@
+[
+{{#file}}
```

Comment:
let's move the handle bars to task and pass them into the library as parameter? that way it will serve as sample for other download tasks and avoid changes in the library if handle bar has to change.